tuf/data/types.go

```diff
@@ -111,6 +111,7 @@ type Signature struct {
 	KeyID     string       `json:"keyid"`
 	Method    SigAlgorithm `json:"method"`
 	Signature []byte       `json:"sig"`
+	IsValid   bool         `json:"-isvalid"`
```

Comment:
should the json tag for the key just be `-` so it's ignored?  Based on https://golang.org/pkg/encoding/json/#Marshal